.github/workflows/kotlin.yml

```diff
@@ -78,7 +78,7 @@ jobs :
           gradle-dependencies-cache-key : |
             gradle/libs.versions.toml
           arguments : |
-            test apiCheck artifactsCheck dependencyGuard lint ktlintCheck jvmWorkflowNodeBenchmarkJar --no-daemon --stacktrace --continue
+            test apiCheck artifactsCheck dependencyGuard lint ktlintCheck jvmWorkflowNodeBenchmarkJar :workflow-runtime:jvmTest :workflow-core:jvmTest --no-daemon --stacktrace --continue
```

Comment:
Perhaps we could replace the `test` task with `check`?

Reply:
Managed to get a red-green on CI with `check` here: https://github.com/square/workflow-kotlin/pull/806